.github/ISSUE_TEMPLATE/documentation.yaml

```
@@ -0,0 +1,40 @@
+name: Documentation
+description: Write a report about the issue
+title: "[documentation] <title>"
+labels: "documentation"
+
+body:
+- type: input
+  attributes:
+    label: Title
+    description: title of documentation
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Does the document's page already exist? 
+    description: Please leave the link if a document's page already exists.
```

For description of here. if there is an example, it will be helpful for issuers

```suggestion
    description: Please link to it. (Example: https://github.com/euphony-io/euphony/blob/master/CONTRIBUTING.md)
```